inspec/a2-iam-v2-no-legacy-policies/controls/role_access.rb

```diff
@@ -41,43 +56,84 @@
           password: ENV['AUTOMATE_API_DEFAULT_PASSWORD'] || 'chefautomate',
         }.to_json
       )
-
-      puts create_user_resp.parsed_response_body
       expect(create_user_resp.http_status).to eq 200
 
+      create_policy_resp = automate_api_request("/apis/iam/v2beta/policies",
```

Comment:
we could also add the viewer and editor as a members of their respective chef-managed policies

Reply:
Ah that would probably be better!